aio/deploy/helm-chart/kubernetes-dashboard/templates/ingress.yaml

```diff
@@ -0,0 +1,67 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- if .Values.ingress.enabled -}}
```

Comment:
```suggestion
{{ if .Values.ingress.enabled -}}
```

Without this, the rendered template looks like this:

```
# See the License for the specific language governing permissions and
# limitations under the License.apiVersion: extensions/v1beta1
kind: Ingress
metadata:
```

and helm complains about a missing `apiVersion`.

Reply:
Indeed. Adding the licence broke all those files. I'll work to have better testing.